test/modules/geo-layers/tile-layer/tile-cache.spec.js

```diff
@@ -3,6 +3,10 @@ import TileCache from '@deck.gl/geo-layers/tile-layer/utils/tile-cache';
 import Tile from '@deck.gl/geo-layers/tile-layer/utils/tile';
 import {WebMercatorViewport} from '@deck.gl/core';
 
+/* eslint-disable no-console, no-undef */
+const onError = console.error;
```

Comment:
There should not be any console error log from tests. If errors are expected let's test for that. Otherwise they need to throw.